fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -369,7 +369,6 @@ def __switch_change_handlers(self, lst, event, handler):
             handlers.__iadd__(handler)
 
         def set_as_primary(self):
-            self.__panel.get_parent().reorder_child(self.__panel, 0)
```

Comment:
Removing this seems to have broken the panel switcher.  I can no longer switch the 1st and 2nd panels around using the double arrow button.

Reply:
Fixed by reverting to a Box object. Test updated as well.